coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "line": 87.08
+    "line": 86.43
```

Comment:
So, this fixes the Travis failure, but I would prefer not to have a coverage drop.

A simple test or two of `Passenger.to_csv` should actually take care of it?